plugins/woocommerce-docs/package.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "woocommerce-docs",
+	"version": "1.0.0",
+	"description": "WooCommerce Docs Plugin",
+	"main": "index.js",
+	"scripts": {
+		"test": "echo \"Error: no test specified\" && exit 1",
+		"build": "wp-scripts build"
+	},
+	"keywords": [],
+	"author": "",
+	"license": "GPL-3.0+",
+	"devDependencies": {
+		"@types/react": "^17.0.2",
+		"@types/react-dom": "^17.0.2",
+		"@woocommerce/dependency-extraction-webpack-plugin": "workspace:*",
+		"@woocommerce/eslint-plugin": "workspace:*",
+		"@wordpress/api-fetch": "wp-6.0",
+		"@wordpress/prettier-config": "^2.16.0",
+		"@wordpress/scripts": "^26.4.0",
+		"eslint": "^8.32.0",
+		"prettier": "npm:wp-prettier@^2.6.2",
+		"react": "^17.0.2",
```

Comment:
wouldn't this be a regular dependency?

Reply:
@psealock I'm conflicted. I don't think anything is a production dependency since we build/bundle the assets and deliver them in production. Even moreso for React because we don't even bundle React, we depend on the React loaded by `@wordpress/element`.

That said for consistency or some kind of removal of confusion I'm happy to make this change.

Reply:
Done here https://github.com/woocommerce/woocommerce/pull/38406/commits/98fc246301a36f98db1d70d36dd6507e0ff2959a